docs/custom-plugins/custom-decorators.md

```diff
@@ -44,37 +47,37 @@ To give a small (but fun) example, here is a decorator that adds a sparkle emoji
 To help keep the plugin code organized, this example uses one file per decorator. In this example, this is the file `plugins/decorators/operation-sparkle.js`:
 
 ```js
-module.exports = OperationSparkle;
-
-function OperationSparkle() {
+export function OperationSparkle() {
   console.log("adding sparkles ... ");
   return {
     Operation: {
       leave(target) {
-        if(target.description) {
+        if (target.description) {
           target.description = "✨ " + String(target.description);
         }
-      }
+      },
     },
-  }
-};
+  };
+}
 ```
 
 Decorators use the [visitor pattern](./visitor.md) to run an operation on every node in the document. In this example, when the code executes the `leave()` function on the `Operation` node, it checks if the node (passed as `target` in this example) has a description, and updates it if it does.
 
 To use this decorator, add it to a plugin. In this example the main decorator file is `plugins/sparkle.js`:
 
 ```js
-const OperationSparkle = require('./decorators/operation-sparkle.js');
-
-module.exports = {
-  id: 'sparkle',
-  decorators: {
-    oas3: {
-      'operation-sparkle': OperationSparkle,
-    }
-  }
-};
+import { OperationSparkle } from './decorators/operation-sparkle.js';
+
+export default function sparklePlugin() {
```

Comment:
Same question, what should the function be named or where is the function name used? Can I export more than one function?

Reply:
The function name does not matter but it's required by the javascript syntax.

Reply:
No, you can only export one function just like you could only export one object, no changes in this regard.